src/firejail/main.c

```diff
@@ -982,6 +982,14 @@ int main(int argc, char **argv, char **envp) {
 	int arg_caps_cmdline = 0; 	// caps requested on command line (used to break out of --chroot)
 	char **ptr;
 
+#ifndef HAVE_SUID
+	if (geteuid() != 0) {
+		fprintf(stderr, "Error: Firejail needs to be SUID.");
```

Comment:
You also want to add newlines (`\n`), as otherwise the error message looks a bit weird (everything in one line).

Reply:
Absolutely right.